src/libraries/Common/src/Interop/Windows/Activeds/Interop.ErrorCodes.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+internal static partial class Interop
+{
+    internal static partial class Activeds
+    {
+        public const int ERROR_SUCCESS = 0;
+        public const int ERROR_MORE_DATA = 234;
+    }
```

Comment:
The error codes are already defined at https://github.com/dotnet/runtime/blob/2959e1acd09aec0f1e59db9cbe41f3701b0bd4dd/src/libraries/Common/src/Interop/Windows/Interop.Errors.cs

Reply:
Ahah, thanks! Fixed in 3a754919e4cff17ea0e14062a1d1ebfd3647ce4b